calico_test/tests/st/utils/docker_host.py

```diff
@@ -132,7 +136,9 @@ def start_calico_node(self, options=""):
         :param as_num: The AS Number for this node.  A value of None uses the
         inherited default value.
         """
-        args = ['node', '--ip=%s' % self.ip]
+        args = ['node']
+        if self.ip:
+            args.append('--ip=%s' % self.ip)
         try:
             if self.ip6:
                 args.append('--ip6=%s' % self.ip6)
```

Comment:
Ugh - I stand corrected - it looks like we still don't initialize the ip6 in the dind branch - can we just make sure we always assign ip6, so we don't have to catch an attribute error.